flake.nix

```diff
@@ -12,8 +12,8 @@
   };
 
   outputs = { self, nixpkgs, ... }@inputs: inputs.utils.lib.eachSystem [
-    "x86_64-linux" "i686-linux" "aarch64-linux" "x86_64-darwin"
+    "x86_64-linux"
   ] (system: {
-    devShell = inputs.alf-devenv.devShell."${system}";
+    devShell = inputs.alf-devenv.devShells."${system}".default;
```

Comment:
will this change affect some of the documentation?

Reply:
Nope this will not.